journal-of-paleontology.csl

```diff
@@ -152,10 +152,10 @@
                 <text variable="page"/>
               </else>
             </choose>
-            <text term="page" form="short"/>
+            <text term="page" form="short" prefix="p. "/>
```

Comment:
this isn't right. The `term="page"` already gives you `p.` or `pp.` so you definitely don't want a `p. ` prefix on it. What's the intended end-result here?

Reply:
This was an error. Thanks for spotting it.